scripts/install_deepchem_conda.ps1

```diff
@@ -20,23 +20,17 @@ else
 }
 
 conda install -y -q -c deepchem -c rdkit -c conda-forge -c omnia `
+    biopython `
     mdtraj `
-    pdbfixer `
-    rdkit `
-    joblib `
-    scikit-learn `
     networkx `
+    openmm `
+    pdbfixer `
     pillow `
-    pandas `
```

Comment:
Don't we still need pandas as a dependency?

Reply:
hard dependencies (pandas and scikit-learn) are installed by `install_requires` in setup.py.
I thinks `install_requires` are better for managing hard dependencies. 

If you use `install_requires`, we don't need to install hard dependencies manually. 
(Current deepchem published by pypi requires manual installations)

Currently, we should install manually like this

```
pip install pandas pillow scikit-learn==0.22 tensorflow-gpu==1.14 deepchem==2.2.1.dev54
```

But after this modification, 

```
pip install tensorflow==2.2 deepchem==2.4.0
```